modules/dnn/src/ie_ngraph.cpp

```diff
@@ -445,6 +486,7 @@ NgraphBackendWrapper::NgraphBackendWrapper(int targetId, const cv::Mat& m)
     , host((Mat*)&m)
 {
     blob = wrapToNgraphBlob(m);
+    hostMatDepth = m.depth();
```

Comment:
unused variable

Reply:
It as not a local variable, it is a field of BackendWrapper class